ui/components/app/confirm-page-container/confirm-page-container.component.js

```diff
@@ -351,7 +373,7 @@ export default class ConfirmPageContainer extends Component {
               <ErrorMessage errorKey={errorKey} />
             </div>
           )}
-          {isSetApproveForAll && isApprovalOrRejection && (
+          {!isSetApproveForAll && isApprovalOrRejection && (
```

Comment:
We should delete this whole block... This appears to just leave a weird piece of dead/unreachable code

Reply:
Removed